### PR TITLE
Fix 4892 - Change Page Title when showing Purchase Modal

### PR DESCRIPTION
**Pull request by Spid3rrr**
Close: #4892 
## PR Details

The issue pertained to the page title not changing when the user clicked purchase and displayed the purchase modal. I used the PageContent existing layout to change the title when the modal was displayed. An alternative solution was to call the Helmet component directly but I thought this way is cleaner.

## PR Checklist

- No tests needed for the change.
- No errors from existing tests.

### CODE REVIEW

**Comment by EugeneTseitlin** [comment]
@Spid3rrr Please, merge changes from amplication:next branch to your branch.

**Review by EugeneTseitlin — CHANGES_REQUESTED** [review]
Please, merge changes from amplication:next branch to your branch.

**Comment by Spid3rrr** [comment]
Will do

________________________________

**Comment by Spid3rrr** [comment]
Should be merged and good to go

**Review by EugeneTseitlin — CHANGES_REQUESTED** [review]
The only change that is expected here is adding a "Helmet" component with a proper text. Please, revert the rest of the changes.

File: packages/amplication-client/src/Purchase/PurchasePage.tsx [hunk 1]
```diff
@@ -119,9 +120,12 @@ const PurchasePage = (props) => {
     },
     [upgradeToPro, handleContactUsClick]
   );
-
+  const pageTitle = "Pricing & Plans";
   return (
     <Modal open fullScreen>
+      <Helmet>
+        <title>{`Amplication${pageTitle ? ` | ${pageTitle}` : ""}`}</title>
```
Comment (EugeneTseitlin) [inline packages/amplication-client/src/Purchase/PurchasePage.tsx:127]: Page title is const, the is no need for validation.

Comment (Spid3rrr) [reply packages/amplication-client/src/Purchase/PurchasePage.tsx:127]: Oh sure thing ! I will revert other changes, remove the validation and commit

**Review by Spid3rrr — COMMENTED** [review]
The changes now should only include the Purchase Page and remove the unnecessary const check.

**Review by EugeneTseitlin — APPROVED** [review]

**Pull request merged**